forge-gui/res/cardsfolder/upcoming/filigree_vector.txt

```diff
@@ -0,0 +1,13 @@
+Name:Filigree Vector
+ManaCost:3 W
+Types:Artifact Creature Phyrexian Construct
+PT:1/1
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigCounter | TriggerDescription$ When CARDNAME enters the battlefield, put a +1/+1 counter on each of any number of target creatures and a charge counter on each of any number of target artifacts.
+SVar:TrigCounter:DB$ PutCounter | ValidTgts$ Creature | TgtPrompt$ Select any number of target creatures to distribute +1/+1 counters to | CounterType$ P1P1 | CounterNum$ X | TargetMin$ 0 | TargetMax$ X | DividedAsYouChoose$ X | SubAbility$ DBChargeCounter
+SVar:DBChargeCounter:DB$ PutCounter | ValidTgts$ Artifact | TgtPrompt$ Select any number of target artifacts to distribute charge counters to | CounterType$ CHARGE | CounterNum$ Y | TargetMin$ 0 | TargetMax$ Y | DividedAsYouChoose$ Y
```

Comment:
It's not divided as you choose if it doesn't say so
You just target and put counters on them 🤷